packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/simulationservice/SimulationRequestResource.java

```diff
@@ -91,4 +99,31 @@ public Simulation makeCalibrateJob(
 		JsonNode jn = Converter.convertObjectToSnakeCaseJsonNode(sim);
 		return simulationProxy.createSimulation(jn);
 	}
+
+	@POST
+	@Path("/calibrate")
+	@Produces(MediaType.APPLICATION_JSON)
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Tag(name = "Create calibrate job")
+	public Simulation makeCalibrateJob(
+		final CalibrationRequest request
+	) {
+		final JobResponse res = simulationServiceProxy.makeCalibrateJob(Converter.convertObjectToSnakeCaseJsonNode(request));
+
+		Simulation sim = new Simulation();
+		sim.setId(res.getSimulationId());
+		sim.setType("simulation");
+
+		sim.setExecutionPayload(request);
+
+		// // FIXME: These fiels are arguable unnecessary
```

Comment:
If they're not needed do we want to remove these, or should they stick around?

Reply:
im not sure what those fields do, just copied and pasted them from the other endpoints. i hear @YohannParis might know a bit more what they do and if they should be removed or not

Reply:
TDS should really handle this portion, but there is some confusion with with @YohannParis want terarium to do??
We will sort it out after the hackathon.